src/messages/search/AuthorPredicate.cpp

```diff
@@ -4,16 +4,14 @@
 
 namespace chatterino {
 
-AuthorPredicate::AuthorPredicate(const QStringList &authors)
-    : authors_()
+AuthorPredicate::AuthorPredicate(const QString &authors, bool negate)
+    : MessagePredicate(negate)
+    , authors_()
 {
     // Check if any comma-seperated values were passed and transform those
-    for (const auto &entry : authors)
```

Comment:
this outer loop was dropped with change of `QStringList` param to `QString`. Previously searchPopup would bundle predicates of same type into one list, but since now each one can be separately negated, it can't be kept.
From user standpoint this changes nothing, comma separated lists typed into search will work the same.